__tests__/articles.test.js

```diff
@@ -81,3 +81,44 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  it("200: should return an array of comments matching the given article_id parameter", () => {
+    const test_id = 1;
+    return request(app)
+      .get(`/api/articles/${test_id}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
```

Comment:
One missing test and behaviour - Status 200, valid ID, but has no comments responds with an empty array of comments.  
You will need to check the reason for the query returning empty rows:
article exists but has no comments.
article does not exist.
You will need to check whether the article exists in order to determine this.